app/services/moab_validation_handler.rb

```diff
@@ -64,6 +64,13 @@ def update_status(new_status)
     complete_moab.status_details = results.results_as_string(results.result_array)
   end
 
+  def mark_moab_not_found
+    results.add_result(AuditResults::MOAB_NOT_FOUND,
+                       db_created_at: complete_moab.created_at.iso8601,
+                       db_updated_at: complete_moab.updated_at.iso8601)
+    update_status('online_moab_not_found')
+  end
```

Comment:
lovin' this refactor

Reply:
i was a bit hesitant to further embrace a module pattern we're not wild about in retrospect, but this did feel more right than copypasta-ing the same code into a new place, since refactoring away from the module isn't on the table atm.